catkin_ws/src/control/src/control_node.cpp

```diff
@@ -9,19 +9,31 @@ std_msgs::String motorString;
 
 // Create your callback function for your publisher that takes in the std_msgs::Int32 message as a ConstPtr,
 // sets motorString.data, and publishes motorString.
+void moveCallback(const std_msgs::Int32::ConstPtr &x_msg) {
+    int x = x_msg->data;
 
+    if (x < 620) {
+        motorString.data = "left";
+    } else if (x > 660) {
+        motorString.data = "right";
+    } else {
+        motorString.data = "forward";
+    }
 
-int main(int argc, char** argv)
-{
+    motor_pub.publish(motorString);
+}
+
+int main(int argc, char **argv) {
     ros::init(argc, argv, "control");
     ros::NodeHandle nh;
-    // Declare the motor publisher here that publishes a std_msgs::String to "/jetbot_motors/cmd_str"
-    
+    motor_pub = nh.advertise<std_msgs::String>("/jetbot_motors/cmd_str", 10);
```

Comment:
Any reasoning behind a message queue size of 10?

Reply:
I thought that if I were to send a stop message while python was backlogged that the robot would never receive that stop message. For something like this it prob makes zero difference.